third_party/accessibility/ax/ax_position.h

```diff
@@ -1935,6 +1951,12 @@ class AXPosition {
           parent_affinity = ax::mojom::TextAffinity::kDownstream;
         }
 
+        AXPositionInstance dummy_position =
+            CreateTextPosition(tree_id, parent_id, 0, parent_affinity);
+        max_text_offset_in_parent = dummy_position->MaxTextOffset();
+        if (parent_offset > max_text_offset_in_parent) {
+          parent_offset = max_text_offset_in_parent;
+        }
```

Comment:
From my understanding, this is new code that's not from Chromium? Are we okay with `third_party/accessiblity` drifting away from Chromium's implementation? Should we try to keep our fork somewhat in sync with Chromium? /cc @chunhtai

Reply:
I am ok drifting away from chromium. I don't think it is possible to keep them in sync in the long run.